crates/bevy_mesh/src/primitives/dim3/plane.rs

```diff
@@ -13,12 +13,23 @@ pub struct PlaneMeshBuilder {
     ///
     /// 0 - is the original plane geometry, the 4 points in the XZ plane.
     ///
-    /// 1 - is split by 1 line in the middle of the plane on both the X axis and the Z axis, resulting in a plane with 4 quads / 8 triangles.
+    /// 1 - adds a vertex in the middle of the X axis, resulting in a plane with 2 quads / 4 triangles, and a new edge along the Z axis.
     ///
-    /// 2 - is a plane split by 2 lines on both the X and Z axes, subdividing the plane into 3 equal sections along each axis, resulting in a plane with 9 quads / 18 triangles.
+    /// 2 - adds 2 vertices along the X axis, resulting in a plane with 3 quads / 6 triangles.
     ///
     /// and so on...
-    pub subdivisions: u32,
+    pub subdivisions_x: u32,
+
+    /// The number of subdivisions in the mesh.
```

Comment:
```suggestion
    /// The number of subdivisions in the mesh along the Z axis.
```

Reply:
and same for `subdivisions_x`, but I can't suggest code change on it as it's technically not changed by this pr